pkg/tpm/tpm.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015 CoreOS, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build tpm
```

Comment:
I suppose that the libs detection could be added in [Miscellaneous stuff section in configure.ac](https://github.com/coreos/rkt/blob/master/configure.ac#L379). If the required libraries exist, set some variable to `-tags tpm` and `AC_SUBST` it in [SUBSTITUTIONS section in configure.ac](https://github.com/coreos/rkt/blob/master/configure.ac#L477). Then add the substituted value to [RKT_TAGS in variables.mk.in](https://github.com/coreos/rkt/blob/master/makelib/variables.mk.in#L21).

And no, we do not document build tags. Should we?